.github/workflows/ci.yaml

```diff
@@ -3,13 +3,14 @@
 name: Build and publish Docker image
 
 env:
-  POSTGRES_VERSION: "15.3"
+  POSTGRES_VERSION: "15.5"
   PGVECTOR_VERSION: "0.4.4"
```

Comment:
Does the PGVECTOR_VERSION need to change here? I think the most recent available version is `0.5.1`

Reply:
we'll do that separately next week when we're ready to upgrade pgvector. for now, we're just upgrading postgres